tsconfig.json

```diff
@@ -23,6 +23,6 @@
       "@/hooks/*": ["hooks/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "cms/index.js"],
```

Comment:
can we just change cms/index to typescript and avoid changing the ts config?

Reply:
Done